docs/organizations/invitations.md

```diff
@@ -14,10 +14,24 @@ Invitees may or may not be existing users. If you choose to allow invitations to
 
 It is possible to pre-select any roles for a user by passing the `roles` array in the API (pre-selecting in the Admin UI is currently not implemented). This will automatically add the user to the given roles upon user creation and acceptance of the invitation.
 
-Invitees will receive an email indicating the realm, organization and inviter. It is possible to add a specific application redirect URI so that they will be redirected to a specific location in the application.
+Make to adapt the [Authentication](https://github.com/p2-inc/keycloak-orgs?tab=readme-ov-file#authentication) according to your needs. Note email verification must be enabled.
```

Comment:
I think just "Adapt the Authentication Flow..." as "Make to adapt" doesn't make sense to me.

Also, clarify why email verification is required. E.g. "Note: Users must have a verified email address to accept an invitation, as it presents a security issue to allow any User to register with an unverified email address and claim the invitation."

Reply:
updated as per your suggestions